dockorm/container.py

```diff
@@ -176,12 +176,10 @@ def port_bindings(self):
     def open_container_ports(self):
         out = []
         for key in self.ports:
-            if isinstance(key, int):
+            if isinstance(key, (int, tuple)):
```

Comment:
What's the tuple case here?

Reply:
It's the case of [specifying a protocol as well](https://github.com/quantopian/DockORM/blob/0af6ff546c0591514761439821420502c22de0ad/dockorm/tests/test_container.py#L185). Previously, the API didn't want the protocol passed here, but at some point that changed to require it, so now we pass it through.

From https://docker-py.readthedocs.io/en/stable/api.html:
> If you wish to use UDP instead of TCP (default), you need to declare ports as such in both the config and host config:
```python
container_id = cli.create_container(
    'busybox', 'ls', ports=[(1111, 'udp'), 2222],
    host_config=cli.create_host_config(port_bindings={
        '1111/udp': 4567, 2222: None
    })
)
```